components/wrap-collection-layout.js

```diff
@@ -47,13 +49,9 @@ function WrapCollectionComponent(Component) {
       associatedMetadata: associatedMetadata,
     });
 
-    const maybeWrapLoadMore = (component, {enable_load_more_button = false}) => {
-      return !enable_load_more_button ?
-        component :
-        addLoadMore(component, data, props.collection.slug, associatedMetadata.subsequent_stories_load_count);
-    }
-    
-    return [maybeWrapLoadMore, maybeWrapClientSide, maybeWrapLazyLoad].reduce((c, f) => f(c, associatedMetadata), Component);
+    const component = maybeWrapLoadMore(Component, data, associatedMetadata.enable_load_more_button, props.collection.slug, associatedMetadata.subsequent_stories_load_count);
+
+    return [maybeWrapClientSide, maybeWrapLazyLoad].reduce((c, f) => f(c, associatedMetadata), component);
```

Comment:
Rename args for better readability

Reply:
Me and Amith thought of a few names actually and finally fixed these. Can you suggest please ?